components/spreadsheet/functions-formulas.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Functions and Formulas
+page_title: Spreadsheet - Functions and Formulas
+description: Learn about all built-in functions supported by the Telerik UI for Blazor Spreadsheet.
+slug: spreadsheet-functions-formulas
+tags: telerik,blazor,spreadsheet
+published: True
+position: 30
+---
+
+# Spreadsheet Functions and Formulas
+
+This article lists the supported built-in functions and formulas by Telerik Spreadsheet for Blazor.
+
+
+## Formulas
+
+A Spreadsheet formula starts with the equal sign `=` and can include:
+
+* References to cells and ranges
+* Constants and literals
+* Operators
+* [Functions](#function-list)
+
+For more information, you can refer to:
+
+* [Microsoft Formula documentation](https://support.microsoft.com/en-us/office/overview-of-formulas-in-excel-ecfdc708-9162-49e8-b993-c311f47ca173)
+* [Microsoft Operator documentation](https://support.microsoft.com/en-gb/office/calculation-operators-and-precedence-in-excel-48be406d-4975-4d31-b2b8-7af9e0e2878a)
+
+To use `=` as a string at the beginning of a cell, start with an apostrophe: `'=`.
+
+
+## Function List
+
+The Telerik Spreadsheet supports a large variety of functions. They work in the same way as in other Excel editors, so the following documentations are also applicable:
+
+* [Microsoft Excel functions](https://support.microsoft.com/en-us/office/excel-functions-alphabetical-b3944572-255d-4efb-bb96-c6d90033e188)
```

Comment:
Consider applying the suggestions from the previous list with links to Microsoft's docs.